config/prod/prow/jobs/config.yaml

```diff
@@ -8819,6 +8819,39 @@ periodics:
     - name: nightly-account
       secret:
         secretName: nightly-account
+- cron: "49 */2 * * *"
+  name: ci-knative-sandbox-net-certmanager-latest
+  agent: kubernetes
+  decorate: true
+  cluster: "build-knative"
+  extra_refs:
+  - org: knative-sandbox
+    repo: net-certmanager
```

Comment:
Just want to double check the repo is net-certmanager not net-istio? (as in the description of this PR, it mentions net-istio)

Reply:
Ah it looks it was added at the wrong place... @tcnghia could you confirm please?